_dashboards/get-started/dql.md

```diff
@@ -0,0 +1,178 @@
+---
+layout: default
+title: Using Dashboards Query Language
+parent: Understanding the core concepts of OpenSearch Dashboards
+nav_order: 40
+redirect_from:
+  - /dashboards/dql/
+---
+
+# Using Dashboards Query Language
+
+Dashboards Query Language (DQL) is a simple text-based query language for filtering data in OpenSearch Dashboards. Similar to [Query DSL]({{site.url}}{{site.baseurl}}/opensearch/query-dsl/index), DQL uses an HTTP request body. For example, to display your site visitor data for a host in the United States, you would enter `geo.dest:US` in the search field, as shown in the following image.
+
+<img src="{{site.url}}{{site.baseurl}}/images/dashboards/dql-interface.png" alt="Search term using DQL toolbar in Dashboard" width="500">
+
+Before you can search data in Dashboards, you must index it. In OpenSearch, the basic unit of data is a JSON document. Within an index, OpenSearch identifies each document using a unique ID. To learn more about indexing in OpenSearch, see [Index data]({{site.url}}{{site.baseurl}}/opensearch/index-data).
+{: .note purple}
+
+## Searching with terms queries
+
+The most basic query specifies the search term:
+
+```
+host:www.example.com
+```
+{% include copy.html %}
+
+To access an object's nested field, list the complete path to the field separated by periods. For example, use the following path to retrieve the `lat` field in the `coordinates` object:
+
+```
+coordinates.lat:43.7102
+```
+{% include copy.html %}
+
+DQL supports leading and trailing wildcards, so you can search for any terms that match your pattern.
+
+```
+host.keyword:*.example.com/*
+```
+{% include copy.html %}
+
+To check whether a field exists or has any data, use a wildcard to see whether Dashboards returns any results:
+
+```
+host.keyword:*
+```
+{% include copy.html %}
+
+## Searching with Boolean queries
+
+To mix and match or combine multiple queries for more refined results, you can use the Boolean operators `and`, `or`, and `not`. DQL is not case sensitive, so `AND` and `and` are the same.
+
+```
+host.keyword:www.example.com and response.keyword:200
```

Comment:
Reference/introduce the example in the text that precedes it.

Reply:
Change made.